debezium-api-generator/src/main/java/io/debezium/api_generator/OpenApiGenerator.java

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Should we call it "schema genarator"? It's not only about creating an OpenAPI or JSON Schema. (waiting for an XSD SPI :smile:)